src/main/scala/uk/gov/hmrc/wco/dec/Response.scala

```diff
@@ -0,0 +1,253 @@
+
+case class ResponseDateTimeElement(
+  @JacksonXmlProperty(localName = "DateTimeString", namespace = NS.rs)
+  dateTimeString: DateTimeString
+)
+
+case class ResponseAdditionalInformation(
+  @JacksonXmlProperty(localName = "StatementCode", namespace = NS.res)
+  statementCode: Option[String] = None, // max 17 chars
+
+  @JacksonXmlProperty(localName = "StatementDescription", namespace = NS.res)
+  statementDescription: Option[String] = None, // max 512 chars
+
+  @JacksonXmlProperty(localName = "LimitDateTime", namespace = NS.res)
+  limitDateTime: Option[ResponseDateTimeElement] = None, // max 35 chars
+
+  @JacksonXmlProperty(localName = "StatementTypeCode", namespace = NS.res)
+  statementTypeCode: Option[String] = None, // max 3 chars
+
+  @JacksonXmlProperty(localName = "Pointer", namespace = NS.res)
+  pointers: Seq[ResponsePointer] = Seq.empty
+)
+
+case class ResponsePointer(
+  @JacksonXmlProperty(localName = "SequenceNumeric", namespace = NS.res)
+  sequenceNumeric: Option[Int] = None, // min 0 max 99999
+
+  @JacksonXmlProperty(localName = "DocumentSectionCode", namespace = NS.res)
+  documentSectionCode: Option[String] = None, // max 3 chars
+
+  @JacksonXmlProperty(localName = "TagID", namespace = NS.res)
+  tagId: Option[String] = None // max 4 chars
+)
+
+case class ResponseAmendment(
+  @JacksonXmlProperty(localName = "ChangeReasonCode", namespace = NS.res)
+  changeReasonCode: Option[String] = None, // max 3 chars
+
+  @JacksonXmlProperty(localName = "Pointer", namespace = NS.res)
+  pointers: Seq[ResponsePointer] = Seq.empty
+)
+
+case class Response(
+  @JacksonXmlProperty(localName = "FunctionCode", namespace = NS.res)
+  functionCode:Int, // max length 2 pattern ([0-9])*"
+
+  @JacksonXmlProperty(localName = "FunctionalReferenceID", namespace = NS.res)
+  functionalReferenceId: Option[String] = None, // max 35 chars
+
+  @JacksonXmlProperty(localName = "IssueDateTime", namespace = NS.res)
+  issueDateTime: Option[ResponseDateTimeElement] = None,
+
+  @JacksonXmlProperty(localName = "AdditionalInformation", namespace = NS.res)
+  additionalInformations: Seq[ResponseAdditionalInformation] = Seq.empty,
+
+  @JacksonXmlProperty(localName = "Amendment", namespace = NS.res)
+  amendments: Seq[ResponseAmendment] = Seq.empty,
+
+  @JacksonXmlProperty(localName = "AppealOffice", namespace = NS.res)
+  appealOffice: Option[AppealOffice] = None,
+
+  @JacksonXmlProperty(localName = "Bank", namespace = NS.res)
+  bank: Option[Bank] = None,
+
+  @JacksonXmlProperty(localName = "ContactOffice", namespace = NS.res)
+  contactOffices: Seq[ContactOffice] = Seq.empty,
+
+  @JacksonXmlProperty(localName = "Error", namespace = NS.res)
+  errors: Seq[Error] = Seq.empty,
+
+  @JacksonXmlProperty(localName = "Status", namespace = NS.res)
+  status: Seq[Status] = Seq.empty,
+
+  @JacksonXmlProperty(localName = "Declaration", namespace = NS.res)
+  declaration:Option[ResponseDeclaration] = None
+)
+
+case class AppealOffice(
```

Comment:
It might be worth renaming as `ResponseAppealOffice` to be explicit about child entities of `Response` and protect against potential package namespace clashes.